pkgs/development/python-modules/pontos/default.nix

```diff
@@ -17,7 +17,7 @@
 
 buildPythonPackage rec {
   pname = "pontos";
-  version = "23.3.5";
+  version = "23.3.6";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
```

Comment:
Or 3.9? See https://github.com/greenbone/pontos/compare/v23.3.5...v23.3.6#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R38

Reply:
Thanks for the hint.